magento/inventory
# MC-19743: Cannot add more than 20 sources to my stock in MSI

### Description (*)


Fix for issue with assigning more than 20 sources at once

### Fixed Issues (if relevant)


### Manual testing scenarios (*)

ISSUE:
Cannot add more than 20 sources to my stock in MSI. Error ‘"priority" should be greater then 0.’

STEPS TO REPLICATE:
Add 21 inventory sources 
Try to allocate 21 sources to a single stock and will throw error "priority" should be greater then 0.

EXPECTED RESULTS:
It should successfully assign all the sources selected.

ACTUAL RESULTS:
Error thrown "priority" should be greater then 0.

### Questions or comments


### Contribution checklist (*)
